README.md

```diff
@@ -1,5 +1,13 @@
-This repository contains a list of tests in the Salt Test Suite that are **not** executed. Skipping
-tests is bad and should only be used as a last resort.
+This repository contains a list of tests in the Salt Test Suite that are **not** executed. Remember: **Skipping tests is bad and should only be used as a last resort.**
```

Comment:
[NIT]: Do we need this? Everyone knows it's bad to skip tests :)) 

```suggestion
This repository contains a list of tests in the Salt Test Suite that are **not** executed.
```

Reply:
Of course you're right. We can leave it out.